docs/git-resolver.md

```diff
@@ -146,6 +146,55 @@ spec:
   * BitBucket Server
   * BitBucket Cloud
 
+## `ResolutionRequest` Status
+`ResolutionRequest.Status.Source` field captures the source where the remote resource came from. It includes the 3 subfields: `url`, `digest` and `entrypoint`.
+- `url`
```

Comment:
Is this data enough to uniquely identify the file? Does the branch have to be encoded somewhere?